cloudprovision.json

```diff
@@ -0,0 +1,139 @@
+{
+  "create": {
+    "steps": [
+      {
+        "id": "clone solution repo",
+        "name": "gcr.io/cloud-builders/git",
+        "entrypoint": "bash",
+        "args": [
+          "-c",
+          "git clone -b feature/enable-cloud-provision https://github.com/gitrey/quota-monitoring-solution.git"
+        ]
+      },
+      {
+        "id": "replace tf vars",
+        "name": "ubuntu",
+        "entrypoint": "bash",
+        "env": [
+          "PROJECT_ID=$PROJECT_ID",
+          "_APP_ID=$_APP_ID",
+          "_APP_NAME=$_APP_NAME",
+          "_REGION=$_REGION",
+          "_SERVICE_ACCOUNT=$_SERVICE_ACCOUNT",
+          "_BUCKET_NAME=$_BUCKET_NAME",
+          "_NOTIFICATION_EMAIL=$_NOTIFICATION_EMAIL",
+          "_THRESHOLD=$_THRESHOLD"
+        ],
+        "args": [
+          "-c",
+          "sed -i s/PROJECT_ID/$PROJECT_ID/g quota-monitoring-solution/terraform/example/terraform.tfvars"
+        ]
+      },
+      {
+        "id": "tf init",
+        "name": "hashicorp/terraform:1.0.0",
+        "entrypoint": "sh",
+        "args": [
+          "-c",
+          "cd quota-monitoring-solution/terraform/example && terraform init"
+        ]
+      }
+    ]
+  },
+  "destroy": {
+    "steps": [
+      {
+        "id": "clone templates",
+        "name": "gcr.io/cloud-builders/git",
+        "entrypoint": "bash",
+        "args": [
+          "-c",
+          "git clone -b feature/enable-cloud-provision https://github.com/gitrey/quota-monitoring-solution.git"
+        ]
+      },
+      {
+        "id": "tf destroy",
+        "name": "hashicorp/terraform:1.0.0",
+        "entrypoint": "sh",
+        "args": [
+          "-c",
+          "echo \"cd quota-monitoring-solution/terraform/example && terraform destroy\""
+        ]
+      }
+    ]
+  },
+  "inputs": [
+    {
```

Comment:
This section drives Cloud Workbench UI when user selects and deploys the template.